build/build.sh

```diff
@@ -61,12 +60,23 @@ if [ -n "$TRAVIS_TAG" ]; then
     # Example: v1.10.0-custom maps to 1.10.0-custom
     VERSION="${TRAVIS_TAG#v}"
 else
-    VERSION="dev"
+    ## Counting dots in CURRENT_BRANCH. If it is
+    ## release branch then it will have more than
+    ## one dot(.) because we are following release
+    ## branch naming convention as v1.10.x, v0.9.x
+    ## v2.0.x, v1.0.x
+
+    ## If we compare with master then local building
+    ## will have unknown version
+    dot_count=$(echo "${CURRENT_BRANCH}" | grep -o "\." | wc -l)
+    if [ "${dot_count}" -gt 1 ]; then
```

Comment:
what is the need to do this? why not just do VERSION="${CURRENT_BRANCH#v}-ci" non tagged build?

Reply:
Basically following is the flow when the user creates  PVC:
- CSI driver receives volume create requests and creates CVC(With all required details and it will populate version also -- We need to fix it but I will take in the current release but this PR will go to `kubera`).
- CVC controller will reconcile for CVCs only if `cvc.VersionDetails.Status.Current` matches to VERSION value in CVC controller.

If we follow VERSION value as `${CURRENT_BRANCH#v}-ci` then if any contributor builds the code and tries to test volume provisioning then it will fail due to version mismatch.

Reply:
here in this case also the same problem is there. Say CSI version he is using is 1.11.0 and when he will comile the code locally if
1. the branch is v1.11.x,  version will be 1.11.x-dev
2. the branch is master, then version will be dev 

He has to compile all the dependant repo to test his change?

Reply:
> He has to compile all the dependant repo to test his change?

Yes(Need to remove dependency). I raised an issue https://github.com/openebs/cstor-operators/issues/108 which will be targeted for this release.

Reply:
> the branch is v1.11.x, version will be 1.11.x-dev
I was in a thought that contributor will checkout from release branch and create tags then build it.